CONTRIBUTING.md

```diff
@@ -130,6 +130,24 @@ cypress window will open locally - select test file to run the tests
 
 cypress window will open locally - select test file to run the tests
 
+#### Test UX and SDK changes
+
+For any new change, which involves changing any of the python SDK components and UI components, the manual testing of the code change can be done using the following steps:
+
+1. git clone <https://github.com/microsoft/responsible-ai-toolbox>
+2. `cd responsible-ai-toolbox` (It is recommended to create a new virtual environment and install the dependencies)
+3. You should commit all your current set of changes for SDK and UX using `git commit`.
+4. Clean all untracked files using `git clean -fdx`
+5. Run `yarn install` and `yarn buildall` to build the UX changes.
+6. Run `pip install -e responsibleai` to install responsibleai locally.
+7. Run `pip install -e raiwidgets` to install raiwidgets locally.
+8. Run `pip install -e erroranalysis` to install erroranalysis locally.
+9. Run `pip install -e rai_core_flask` to install rai_core_flask locally.
```

Comment:
Should this (and the other section above which was added in a previous PR) be updated to have the different order where it starts with raiwidgets? Just wondering since @imatiach-msft was worried about the raiwidgets installation overriding the previous ones (as other packages are dependencies of raiwidgets).

Reply:
I tested editable install 'pip install -e responsibleai' and 'pip install -e raiwidgets' and the installation still points to the local directory. So it looks fine to me.